reactjs-films-homework/src/components/Header/LanguageToggler/__tests__/LanguageToggler.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen, fireEvent, cleanup } from '@testing-library/react';
+import LanguageToggler from '../LanguageToggler';
+
+describe('LanguageToggler', () => {
+  afterEach(() => {
+    cleanup();
+  });
+  it('LanguageToggler snapshot', () => {
```

Comment:
redundant LanguageToggler here.
Because when you run the test you can see the following:
'LanguageToggler LanguageToggler snapshot'
first 'LanguageToggler' from 'describe' and the second 'LanguageToggler' from the 'it'
So convention should be:
'LanguageToggler' in 'describe' and 'should render correctly' in 'it'
So that you will have the following output when you run the tests:
'LanguageToggler should render correctly'

Reply:
the redundant LanguageToggler was removed